test/libsolidity/semanticTests/storage/packed_storage_signed.sol

```diff
@@ -9,7 +9,7 @@ contract C {
         returns (uint256 x1, uint256 x2, uint256 x3, uint256 x4)
     {
         a = -2;
-        b = -uint8(a) * 2;
+        b = (0 - uint8(a)) * 2;
```

Comment:
Is this here `0 - twos_complement(-2)` ?

Reply:
`uint8(-2) == type(uint8).max + 1 - 2`